src/tests/e2etest/LocalManagementTests.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace E2eTesting
+{
+
+    [TestFixture, Category("LocalManagement")]
+    public class LocalManagementTests : E2ETest
+    {
+        private static readonly string _componentName = "CommandRunner";
+        private static readonly string _reportedObjectName = "commandStatus";
+        private static readonly string _desiredPath = "/etc/osconfig/osconfig_desired.json";
+
+        public class CommandRunner
+        {
+            public CommandRunnerTests.CommandArguments commandArguments { get; set; }
+        }
+
+        public class CommandRunnerObject
+        {
+            public CommandRunner CommandRunner { get; set; }
```

Comment:
Not enough for DC path. Both RC and DC paths must be tested same as Reported and Desired Twin paths, for each and every module, 100%.